model/execution/DeliveryExecutionManagerService.php

```diff
@@ -298,6 +320,14 @@ public function adjustTimers(array $deliveryExecutions, $seconds)
         return $result;
     }
 
+    /**
+     * @return array|object
```

Comment:
For IDE search and autocompletion it would be useful to specify the interface of the object we expect to return.

Reply:
fixed